### PR TITLE
Change phrasing of drop database dialog

### PR DESCRIPTION
**Pull request by akritianand**
Signed-off-by: Akriti Anand anand.akriti@gmail.com

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by ibennetch** [comment]
Hi @akritianand, thanks for your submission. In the future, it would be helpful if you can use a more descriptive title for your pull request and for the commits. "Update functions.js" or "Issue #13747" aren't very helpful when looking through the git log. Something like "Improve drop table confirmation message" makes a better pull request title.

The bot is upset that you haven't set the Signed-Off-By line which indicates that you agree with the terms of the [Developer Certificate of Origin (DCO)](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). You can read a bit more about adding this in the [CONTRIBUTING document](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md#patches-submission), but it's most easily accomplished by passing the `--signoff` argument to git commit.

File: js/functions.js [hunk 1]
```diff
@@ -2776,16 +2776,28 @@ jQuery.fn.PMA_confirm = function (question, url, callbackFn, openCallback) {
...
-            },
-            open: openCallback,
-            modal: true
-        });
+    if (question.search('DROP TABLE') != -1) {
+        var heading = question.split('execute "')[1].split('"')[0];
+    }
+
+    else if (question.search('DROP DATABASE') != -1) {
+        var heading = question.split('execute "')[1].split('"')[0];
```
Comment (nijel) [inline js/functions.js]: Both these will break with localized messages.

File: js/functions.js [hunk 2]
```diff
@@ -2776,16 +2776,28 @@ jQuery.fn.PMA_confirm = function (question, url, callbackFn, openCallback) {
...
+    }
+
+    else {
+        var heading = PMA_messages.strConfirm;
+    }
+
+    $('<div/>', {'id': 'confirm_dialog', 'title': heading})
+    .prepend(question)
```
Comment (nijel) [inline js/functions.js]: Any reason for changing indentation on existing code?

**Review by nijel — CHANGES_REQUESTED** [review]
Besides things @ibennetch has already mentioned, I think the patch itself is also wrong. Maybe it would be best to include the title as optional parameter for PMA_confirm, so that every caller could specify it instead of relying on parsing of passed text what will break with localized strings.

File: js/functions.js (hunk 1 above)
Comment (akritianand) [reply js/functions.js]: Can you tell me how will these break localised messages? I'm new to this, so sorry for my ignorance.

Comment (nijel) [reply js/functions.js]: They won't contain the "execute" string (and can contain different quote as well). For example in Czech the string will be "Opravdu si přejete provést „%s“?". So split will return just one string and the code will fail.

**Pull request closed without merging**